src/code/dialogs/blockersList.js

```diff
@@ -45,7 +45,11 @@ const BlockerList = WDialog.extend({
     const buttons = {};
     buttons[wX("OK")] = () => {
       this.closeDialog();
-      window.map.fire("wasabeeUIUpdate", { reason: "blockerlist" }, false);
```

Comment:
this line looked like a wrong copy/paste

Reply:
There was a reason I added this, but I can't recall now.